apis/v1alpha2/shared_types.go

```diff
@@ -483,18 +483,21 @@ type AnnotationValue string
 // AddressType defines how a network address is represented as a text string.
 // This may take two possible forms:
 //
-// * A predefined CamelCase string identifier (currently limited to `IPAddress` or `Hostname`)
+// * A predefined CamelCase string identifier (currently limited to `IPAddress` or  `Hostname`)
```

Comment:
This seems to just be adding extra blank space?

```suggestion
// * A predefined CamelCase string identifier (currently limited to `IPAddress` or `Hostname`)
```

Reply:
Woops, fixed!